source/roadmap.html.erb

```diff
@@ -17,66 +17,78 @@ title: Roadmap
+         We're updating Pay to manage not just card payments but other ways to pay as well. This change is important because it means paying users of public sector services will have more options in future, making GOV.UK Pay able to adapt to different needs. More payment options make it simpler for everyone to use public sector services in the way that works best for them, and allows more public services to benefit from using us. We plan to use pay by bank, which could make payments faster and cheaper while keeping them secure.
+        </p>
+        <h3 class="govuk-heading-m">Procuring new card and pay by bank payment suppliers</h3>
+        <p class="govuk-body">
+          We're completing a procurement exercise to get the best card and pay by bank suppliers. This will help to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
         </p>
-        <h3 class="govuk-heading-m">Accessibility</h3>
-        <p class="govuk-body">We're making sure our service can be used by everyone, following the latest Web Content Accessibility Guidelines 2.2 (WCAG 2.2).</p>
-        <h3 class="govuk-heading-m">Data deletion</h3>
-        <p class="govuk-body">We have reviewed our data retention schedules. We continue to make sure personal data is deleted or redacted within 7 years.</p>
-        <h3 class="govuk-heading-m">Database upgrades</h3>
-        <p class="govuk-body">We are upgrading 5 of our main databases in GOV.UK Pay to the latest Postgres version.</p>
         <h2 class="govuk-heading-l">What's next</h2>
-        <h3 class="govuk-heading-m">Recurring payments improvements</h3>
-        <p class="govuk-body">We'll use feedback from services and paying users to improve how recurring payments work.</p>
-        <h3 class="govuk-heading-m">Improving transaction search</h3>
-        <p class="govuk-body">We're going to make it easier to find and manage transactions in the admin tool.</p>
-        <h3 class="govuk-heading-m">Strengthening our security</h3>
-        <p class="govuk-body">We're getting ready for a new security standard called Payment Card Industry Data Security Standard 4.0 to keep users' information safe.</p>
+        <h3 class="govuk-heading-m">Integrate with new card payment suppliers</h3>
+        <p class="govuk-body">
+          Once we’ve procured new card payment suppliers, we’ll integrate them with GOV.UK Pay.
+        </p>
+        <h3 class="govuk-heading-m">Release pay by bank to a small number of public sector services</h3>
+        <p class="govuk-body">
+          We want to integrate Pay with a pay by bank supplier so we can release pay by bank for a small number of public sector services. This will allow paying users to pay for public sector services with their banking app or online banking. We’ll use what we learn from this work to improve the payment type before releasing it to more public sector services.
```

Comment:
```suggestion
          We want to integrate GOV.UK Pay with a pay by bank supplier so we can release pay by bank for a small number of public sector services. This will allow users to pay for public sector services with their banking app or online banking. We’ll use what we learn from this work to improve pay by bank before releasing it to more public sector services.
```